scalafix/rules/src/main/scala/com/dwolla/scrooge/scalafix/AdaptHigherKindedThriftCode.scala

```diff
@@ -0,0 +1,50 @@
+package com.dwolla.scrooge.scalafix
+
+import com.dwolla.scrooge.scalafix.AdaptHigherKindedThriftCode._
+import scalafix.v1
+import scalafix.v1._
+
+import scala.meta._
+
+class AdaptHigherKindedThriftCode extends SemanticRule("AdaptHigherKindedThriftCode") {
+  override def fix(implicit doc: SemanticDocument): Patch =
+    addObjectQualifierForThriftServiceTrait(doc.tree)
+}
+
+object AdaptHigherKindedThriftCode {
+  private def isGeneratedThriftService(t: Type.Name, name: String)
+                                      (implicit doc: SemanticDocument): Boolean = {
+    val info = t.symbol.info
+
+    val nameEndsWithService = name.endsWith("Service")
```

Comment:
I think there are certain thrift services that don't explicitly end in "service". For example, account-settings or webhook-subscriptions. Would this solution work for those services?

Reply:
Good catch. I thought that suffix was added by Scrooge but it looks like it was just our convention. I'll remove that check.